OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -86,15 +86,15 @@ public class ViewportControllerWidget : Widget
 			{ ScrollDirection.Right, new float2(1, 0) },
 		};
 
-		Lazy<TooltipContainerWidget> tooltipContainer;
+		readonly Lazy<TooltipContainerWidget> tooltipContainer;
```

Comment:
IIRC we usually group those `readonly` fields.

Reply:
Did that now.

Reply:
There are more (the onld ones) at the top.